core-blocks/image/edit.js

```diff
@@ -389,6 +363,11 @@ class ImageEdit extends Component {
 }
 
 export default compose( [
+	withBlockEditContext( ( { setFocusedElement } ) => {
```

Comment:
Nice trick, do you think this should be documented or recommended?

Reply:
I did the same for both `BlockEdit` and `Plugin` - it's more work to map context to props, but I like the fact that you don't need to navigate between files to discover how context is injected into a component.

Reply:
See:
https://github.com/WordPress/gutenberg/blob/master/editor/components/block-edit/context.js#L24-L26

Reply:
and https://github.com/WordPress/gutenberg/blob/master/plugins/components/plugin-context/index.js#L17-L19.

Well, we could even create a factory for it :D